statuses.go

```diff
@@ -58,18 +60,79 @@ const (
 
 	// ChannelNotFoundError means the searched for data transfer does not exist
 	ChannelNotFoundError
+
+	// Queued indicates a data transfer request has been accepted, but is not actively transfering yet
+	Queued
+
+	// AwaitingAcceptance indicates a transfer request is actively being processed by the transport
+	// even if the remote has not yet responded that it's accepted the transfer
```

Comment:
```suggestion
	// even if the remote has not yet responded that it's accepted the transfer. Such a state can
	// occur, for example, in a requestor-initiated transfer that starts processing prior to receiving
	// acceptance from the server.
```

or something like that, or even get specific and talk about graphsync pull reuqests; because this one's a bit of a mystery without the kind of clarification you gave in your OP.